src/app/clients/getClientsTableData.ts

```diff
@@ -1,20 +1,25 @@
 import supabase from "@/supabaseClient";
 import { Schema } from "@/database_utils";
-import { Datum } from "@/components/Tables/Table";
 
-export interface ClientsTableRow extends Datum {
-    primaryKey: string;
+export interface ClientsTableRow {
     parcelId: Schema["parcels"]["primary_key"];
-    flaggedForAttention: boolean;
-    requiresFollowUpPhoneCall: boolean;
+    primaryKey: Schema["clients"]["primary_key"];
     fullName: Schema["clients"]["full_name"];
     familyCategory: string;
     addressPostcode: Schema["clients"]["address_postcode"];
-    collectionCentre: string;
-    congestionChargeApplies: boolean;
+    deliveryCollection: {
+        collectionCentre: string;
+        congestionChargeApplies: boolean;
+    };
     packingTimeLabel: string;
-    collectionDatetime: string;
+    collectionDatetime: Schema["parcels"]["collection_datetime"];
     lastStatus: string;
```

Comment:
Similarly, we have a list of status right - should we ensure they are limited to those that we can choose in the client page?